app/components/publishers/vacancies_component.rb

```diff
@@ -46,13 +46,15 @@ def view_applicants(vacancy, card)
     return unless vacancy.enable_job_applications?
     return unless include_job_applications?
 
-    if vacancy.job_applications.any?
-      link = govuk_link_to(I18n.t("jobs.manage.view_applicants", count: vacancy.job_applications.count),
+    if vacancy.job_applications.submitted.any?
```

Comment:
The ticket doesn't make this clear, but the count should include all job applications that don't have a status of draft or withdrawn